app.json

```diff
@@ -14,5 +14,10 @@
     "description": "A simple app to create polls on Rocket.Chat. Use the slash command: /poll [Question?]",
     "implements": [
         "IUIKitInteractionHandler"
+    ],
+    "permissions": [
```

Comment:
This app uses a lot of other permissions like `slashcommand`, `message.read` etc which we need to also declare over here. For a list of all permissions, please refer [here](https://developer.rocket.chat/apps-development/permission-system) and declare ONLY the permissions being used by this app.

PS: Earlier since this app was using apps-engine version < 1.20.0 which is when the permission system was introduced, we didn't have to explicitly declare any such permissions as the apps-engine assigns such apps `default permission`. However now, since our app is going to be using schedular apis which is on 1.23.0 version of apps-engine - we now need to declare all these permissions explicitly

Reply:
Added the permissions. Please let me know if I missed any. Thanks.